CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Change Log
 
-## [4.0.3](https://github.com/sous-chefs/consul/tree/v4.0.3 (2020-07-20)
+## Unreleased
+
+- Disable broken tests and suites and fix client suite
+- Update kitchen.yml to our standards and default to current channel
+- Add testing for Ubuntu 20.04
+- Fix Dokken configuration for CentOS 8 and Ubuntu 16.04
+- Enable enforce_idempotency checks
```

Comment:
Do we want to say we require RHEL 7+ now that the testing of older RHEL is gone / readme is updated